hapi-fees/src/main/java/com/hedera/services/usage/schedule/ScheduleOpsUsage.java

```diff
@@ -42,7 +42,7 @@
 
 public class ScheduleOpsUsage {
 	/* Scheduled transaction ids have the scheduled=true flag set */
-	private static final long SCHEDULED_TXN_ID_SIZE = BASIC_TX_ID_SIZE + BOOL_SIZE;
+	private static final long SCHEDULED_TXN_ID_SIZE = (long) BASIC_TX_ID_SIZE + BOOL_SIZE;
```

Comment:
I think we should change `BASIC_TX_ID_SIZE` and/or `BOOL_SIZE ` to `long` instead

Reply:
Changing this to long affects changes in a lot of places as both are used in multiple places